src/components/Layout/dashboardMenuContainer/DashboardMenuContainer.tsx

```diff
@@ -1,30 +1,42 @@
 import { Grid } from '@mui/material';
 import { userSelectors } from '@pagopa/selfcare-common-frontend/redux/slices/userSlice';
 import { roleLabels } from '@pagopa/selfcare-common-frontend/utils/constants';
+import { useTranslation } from 'react-i18next';
 import { useAppSelector } from '../../../redux/hooks';
 import { partiesSelectors } from '../../../redux/slices/partiesSlice';
 import LogoSubMenu from './components/LogoSubMenu';
 import DashboardSubMenu from './components/DashboardSubMenu';
 
 const DashboardMenuContainer = () => {
+  const { t } = useTranslation();
   const selectedParty = useAppSelector(partiesSelectors.selectPartySelected);
   const user = useAppSelector(userSelectors.selectLoggedUser);
 
   return selectedParty ? (
     <Grid container item direction="row" xs={6} alignContent="center" justifyContent="flex-end">
       <Grid item xs={5}>
         <LogoSubMenu
```

Comment:
It's not required to think to localize text that in the future could be concatenated to other text.
Thus, it's preferable to simplify and let this text asis